app/views/help/index.html.erb

```diff
@@ -1,54 +1,54 @@
 <% content_for :title, "Help using GOV.UK - Help Pages - GOV.UK" %>
 <% content_for :extra_headers do %>
   <meta name="description"
-        content="Find out about GOV.UK, including the use of cookies, accessibility of the site, the privacy notice and terms and conditions of use." />
+        content="Find out about GOV.UK, including the use of cookies, accessibility of the site, the privacy notice and terms and conditions of use."/>
```

Comment:
```suggestion
        content="<%= t('help.index.find_out') %>"/>
```

Reply:
Thanks for adding this commit @MuriloDalRi.  Would you be able to squash it into the relevant previous commit, then I'm happy to approve 🎉

Reply:
Done